version/distro/distro.go

```diff
@@ -77,9 +77,9 @@ func linuxDistro() Distro {
 	case have("/usr/local/bin/freenas-debug"):
 		// TrueNAS Scale runs on debian
 		return TrueNAS
-	case isUDMPro():
-		// UDM-Pro runs on debian
-		return UDMPro
+	case have("/usr/bin/ubnt-device-info"):
```

Comment:
It looks like USG4 does not have this file. I think I'd like this to succinctly list the product families that are supported. That seems better than discussing what exact path(s) to use to support everything because that may take a while; support for other things can be done in later PRs.

Reply:
@sailorfrag One of the challenges I’m facing is that I don’t have the knowledge or access to a broad range of devices to confirm this definitively. I’m assuming you have a USG4? If so, could you share the output of apt list --installed | egrep -i 'udm|ubnt' from your setup? That would help me better understand and address the supported product families.

Reply:
@notheotherben Do you know the exact list of product families which have this binary? UDM for sure, any others?